applications/openshift/kubelet/var_role_master.var

```diff
@@ -0,0 +1,15 @@
+documentation_complete: true
+
+title: 'Configure which node to scan based on role'
+
+description: 'Configure which node to scan based on role'
+
+type: string
+
+operator: equals
+
+interactive: false
+
+options:
+  default: "master"
+  worker: "worker"
```

Comment:
why does the master variable say worker?

Reply:
Let me remove this